react-admin/build.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/env bash
+
+set -e # exit when any command fails
+
+PUBLIC_URL=/react-admin/
```

Comment:
When declaring variables like this, they will only be available in the current file.


To make it available in the scripts, either use:

```
PUBLIC_URL=/react-admin/ make build
```

or 


```
export PUBLIC_URL=/react-admin/

make build
```

The `export` keyword makes it available for all the sub-processes (like `make` and `npm`)


---

I already made this change on `main` branch